docs/questions/visualizations/funnel.md

```diff
@@ -2,12 +2,87 @@
+Funnel charts visualize how a value is broken out by a series of steps, and the percent change between steps.
 
 ![Funnel](../images/funnel.png)
+
+Funnels are commonly used in e-commerce or sales to visualize how many customers are present within each step of a checkout flow or sales cycle.
+
+## Data shape for a funnel chart
+
+To create a funnel in Metabase, you'll need to have a table with at least two columns: one column that contains the funnel's steps, and another column contains the metric you're interested in, like the number of customers at this stage.
+
+Here's the data shape used to create the chart above:
+
+| Step          | Opportunities |
+| ------------- | ------------- |
+| Leads         | 3901          |
+| Qualification | 3714          |
+| Prospecting   | 3231          |
+| Proposal      | 2972          |
+| Negotiation   | 1588          |
+| Closed        | 737           |
+
+If you have more columns in your query results, you can select which columns should be used for the funnel steps and the metric in the [data settings](#funnel-chart-settings).
+
+By default, Metabase will present steps in the same order as they appear in the query result, but you can reorder or hide the steps in the [data settings](#funnel-chart-settings).
+
+## Build a query for a funnel chart
+
+To create a query with a result that has the shape required for a funnel chart, you'll probably need to summarize your data.
+
+If your (original, unaggregated) data already contains a field with the the step for every data point, you can build a simple query with a breakout by steps:
+
+![A query in the query builder used to build a funnel chart](../images/build-a-funnel-query.png)
+
+But often the data for the different stages of the funnel comes from different tables, or you might need to use different filters or aggregation rules for each step. In this case, you can create separate questions for each step, and then use SQL to combine them.
+
+For example, you can create 3 separate [query builder](../query-builder/editor.md) questions, each returning the counts for `Leads`, `Qualification`, and `Proposal` stages. Then you can write a [SQL query](../native-editor/writing-sql.md) that [references those questions](../native-editor/referencing-saved-questions-in-queries.md) and uses `UNION` to combine the results into a single table with the funnel-appropriate shape
+
+```sql
+-- example of a query that retrieves results of questions and combines them with UNION
+
+SELECT 'Leads' as stage, * from {{#120-leads}}
+UNION
+SELECT 'Qualified' as stage, * from {{#121-qualified}}
+UNION
+SELECT 'Prospects' as stage, * from {{#122-prospects}}
+
+```
+
+![Data for the funnel coming from a SQL union](../images/funnel-as-sql.png)
+
+## How to read a funnel chart
```

Comment:
I don't think the choice that we made with % is very intuitive, I have to always double check which % is displayed on the chart